tensorflow/lite/delegates/external/external_delegate.cc

```diff
@@ -29,10 +30,12 @@ struct ExternalLib {
       const char**, const char**, size_t,
       void (*report_error)(const char*))>::type;
   using DestroyDelegatePtr = std::add_pointer<void(TfLiteDelegate*)>::type;
+  struct wchar_codecvt : public std::codecvt<wchar_t, char, std::mbstate_t> {};
+  std::wstring_convert<wchar_codecvt> converter;
 
   // Open a given delegate library and load the create/destroy symbols
   bool load(const std::string library) {
-    void* handle = SharedLibrary::LoadLibrary(library.c_str());
+    void* handle = SharedLibrary::LoadLibrary(converter.from_bytes(library.c_str()).c_str());
```

Comment:
All the newly added code in this file needs to be within the _WIN32 header guard:

eg:
#if defined(_WIN32)
  void* handle = SharedLibrary::LoadLibrary(converter.from_bytes(library.c_str()).c_str());
#else
 void* handle = SharedLibrary::LoadLibrary(library.c_str());
#endif  // defined(_WIN32)

Refer to lite/shared_library.h for details.

Without this update, the PR cannot be merged as the code will fail on non-windows machines (refer to the failing CI build errors for this PR)